doc/source/install/apt.rst

```diff
@@ -7,9 +7,9 @@ Installing |PXC| on Debian or Ubuntu
 Percona provides :file:`.deb` packages for 64-bit versions
 of the following distributions:
 
-* Debian 7 ("wheezy")
 * Debian 8 ("jessie")
-* Ubuntu 12.04 LTS (Precise Pangolin)
+* Debian 9 ("stretch")
+* Debian 10 ("buster")
 * Ubuntu 14.04 LTS (Trusty Tahr)
 * Ubuntu 16.04 LTS (Xenial Xerus)
 * Ubuntu 17.04 (Zesty Zapus)
```

Comment:
Do we provide support for Zesty (17.04) or Aardvark (17.10)?  We don't do any MTR runs with these distros?